wiki/Tournaments/TWC/2011/en.md

```diff
@@ -3,44 +3,43 @@ tags:
 - TWC 2011
 - TWC2011
 ---
-Taiko World Cup 2011
-======================
+# Taiko World Cup 2011
 
-![TWC 2011 logo](logo.png)
+![TWC 2011 Logo](img/logo.png)
 
-The **osu!taiko World Cup 2011** (***TWC #1***) is a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It is part of the 1st installment of the osu!taiko World Cup.
+The **osu!taiko World Cup 2011** (***TWC 2011***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was part of the 1st installment of the osu!taiko World Cup.
```

Comment:
Not sure about the `part of`, since it seems complete (also logo has `The First Half`, so idk).
```suggestion
The **Taiko World Cup 2011** (***TWC 2011***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the first installment of the osu!taiko World Cup.
```

Reply:
I'll take your suggestion instead.